src/pages/products/prairietest/index.tsx

```diff
@@ -0,0 +1,239 @@
+import { BannerCTA } from "../../../components/CallToActionBanner";
+
+import diagramImage from "../../../lib/images/exam-management.png";
+import scannerImage from "../../../lib/images/swipe.png";
+import accommodationsImage from "../../../lib/images/pt-accommodations.png";
+import unifiedViewImage from "../../../lib/images/unified-view.png";
+import rescheduleImage from "../../../lib/images/reschedule.png";
+import proctorImage from "../../../lib/images/assign-proctor.png";
+import schedulerImage from "../../../lib/images/pt-reservation.png";
+import overridesImage from "../../../lib/images/overrides.png";
+import onlineFormatImage from "../../../lib/images/zoom-proctored.png";
+import cbtfFormatImage from "../../../lib/images/cbtf.jpg";
+import byodFormatImage from "../../../lib/images/byodinclass.png";
+
+import styles from "./index.module.scss";
+
+interface FeatureCardProps {
+  image: ImageProps["src"];
+  title: string;
+  children: React.ReactNode;
+}
+
+const FeatureCard: React.FC<FeatureCardProps> = ({
+  image,
+  title,
+  children,
+}) => {
+  return (
+    <article className={classnames("card overflow-hidden", styles.card)}>
+      <Image
+        src={image}
+        alt={title}
+        style={{
+          objectFit: "contain",
+          width: "30%",
+          height: "30%",
+          // display: "flex",
+          // alignItems: "center",
+          // justifyContent: "center",
+          // flexDirection: "column",
+        }}
+      />
+      <div className="card-body">
+        <h5 className="card-title h5">{title}</h5>
+        {children}
+      </div>
+    </article>
+  );
+};
+
+export default function PrairieTest() {
+  return (
+    <React.Fragment>
+      <Head>
+        <title>PrairieTest | PrairieLearn</title>
+      </Head>
+      <PageBanner
+        title="PrairieTest"
+        subtitle="Planning and delivering testing at scale"
+      />
+
+      <div className={classnames("container-fluid py-4")}>
+        <div className="container-md">
+          <div className="row">
+            <Heading>Powerful exam management system</Heading>
+          </div>
+          <div className="row">
+            <Image
+              src={diagramImage}
+              alt="diagram exam management process"
+              style={{
+                maxWidth: "100%",
+                height: "auto",
+              }}
+            />
+          </div>
+        </div>
+      </div>
+
+      <div className={classnames("container-fluid my-5")}>
+        <div className="container-md">
+          <div className="row">
+            <h4 className={classnames("py-3 ", styles.color, styles.center)}>
+              All exam management done in one place!
+            </h4>
+          </div>
+          <div className={styles.grid}>
+            <FeatureCard image={scannerImage} title="Fast check-in process">
+              Supports card scanners providing fast student verification that
+              enables exams access
```

Comment:
Nit, I'd prefer periods in all these cards.

```suggestion
              Supports card scanners providing fast student verification that
              enables exams access.
```